src/scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/rest/BasicAuthenticationFilter.java

```diff
@@ -40,6 +44,12 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         HttpServletResponse httpResponse = (HttpServletResponse) servletResponse;
         List<String> unprotectedEndpointsConfig = healthServerConfiguration.getUnprotectedEndpoints();
 
```

Comment:
May be there is a better way to register URL mappings using FilterBeanRegistration